stat/roc.go

```diff
@@ -104,17 +104,13 @@ func ROC(cutoffs, y []float64, classes []bool, weights []float64) (tpr, fpr, thr
 			fpr[bin] += negWeight
 		}
 	}
-	lastTpr := tpr[bin]
-	lastFpr := fpr[bin]
-	for bin++; bin < len(cutoffs); bin++ {
-		tpr[bin] = lastTpr
-		fpr[bin] = lastFpr
-	}
+	// here bins beyond the maximum value in y are skipped, and the
+	// fpr and tpr are set to zero below by ommission.
```

Comment:
I'd put this after the comment with the for loop.

```
	// Bins beyond the maximum value in y are skipped
	//  and the fpr and tpr elements are left as zero.
```

Reply:
Done (?).

Reply:
LGTM